### PR TITLE
Rm some fields from the meeting actions.

### PR DESCRIPTION
**Pull request by reiterl**
Remove welcome_title, welcome_text, enable_anonymous.
Update the tests.

### CODE REVIEW

**Review by r-peschke — COMMENTED** [review]
You removed the fields also from meeting.update. In the wiki site for meeting.update they are not removed. 
If you are sure they should be removed, please change the description in wiki, too and feel free to merge:-)

**Comment by jsangmeister** [comment]
I think we should wait for some clarification in #754 here

File: openslides_backend/action/actions/meeting/update.py [hunk 1]
```diff
@@ -20,8 +20,6 @@
 from ..meeting.shared_meeting import used_as_default_for_schema_required
 
 meeting_settings_keys = [
-    "welcome_title",
```
[the remark this answers is not in the document]
Comment (reiterl) [reply openslides_backend/action/actions/meeting/update.py]: Revert the update changes.

File: openslides_backend/action/actions/meeting/update.py [hunk 2]
```diff
@@ -135,7 +133,6 @@ class MeetingUpdate(UpdateAction):
             "jitsi_domain",
             "jitsi_room_name",
             "jitsi_room_password",
-            "enable_anonymous",
```
[the remark this answers is not in the document]
Comment (reiterl) [reply openslides_backend/action/actions/meeting/update.py]: Revert the update changes.

File: openslides_backend/action/actions/meeting/update.py [hunk 3]
```diff
@@ -193,15 +190,7 @@ def check_permissions(self, instance: Dict[str, Any]) -> None:
             raise MissingPermission(Permissions.Projector.CAN_MANAGE)
 
         # group D check
-        if any(
```
[the remark this answers is not in the document]
Comment (reiterl) [reply openslides_backend/action/actions/meeting/update.py]: Revert the update changes.

File: tests/system/action/meeting/test_update.py [hunk 4]
```diff
@@ -239,14 +239,14 @@ def test_update_default_projector_to__projector_from_wrong_meeting_error(
 
     def test_update_group_a_no_permissions(self) -> None:
         self.base_permission_test(
-            self.test_models, "meeting.update", {"id": 1, "welcome_title": "Hallo"}
```
[the remark this answers is not in the document]
Comment (reiterl) [reply tests/system/action/meeting/test_update.py]: Revert the update changes.

**Comment by reiterl** [comment]
I talked with Gabriel and Ralf about this issue. Gabriel updated the issue description and the wiki. Only the **meeting.create** should be modified. I reverted the update part.

**Review by r-peschke — APPROVED** [review]

**Review by FinnStutzenstein — APPROVED** [review]

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**